### PR TITLE
Added chdir to task

### PR DESCRIPTION
**Pull request by sascha-andres**
I wanted to run tools in sub directories of the directory Taskfile.yml lives in. For me it is more convenient to provide a directory to run the command(s) in then prefixing each command with `cd <path> && `

### CODE REVIEW

**Comment by sascha-andres** [comment]
A sample file looks like this:

    build:
      cmds:
        - ./run.sh
      chdir: ./build/

File: task.go [hunk 1]
```diff
@@ -40,6 +40,7 @@ type Task struct {
 	Deps      []string
 	Sources   []string
 	Generates []string
+	Chdir     string
```
Comment (andreynering) [inline task.go]: I think this can be called just "dir".

File: task.go [hunk 2]
```diff
@@ -125,13 +126,16 @@ func isTaskUpToDate(t *Task) bool {
...
 
-func runCommand(c string) error {
+func runCommand(c, path string) error {
 	var cmd *exec.Cmd
 	if ShExists {
 		cmd = exec.Command(ShPath, "-c", c)
 	} else {
 		cmd = exec.Command("cmd", "/C", c)
 	}
+	if "" != path {
```
Comment (andreynering) [inline task.go]:
```go
path != ""
```

**Comment by sascha-andres** [comment]
Updated code

**Comment by andreynering** [comment]
Thank you very much

**Pull request merged**